org.syphon.Syphon.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=Syphon
+Icon=org.syphon.Syphon
+Exec=env GDK_GL=gles syphon %U
```

Comment:
this should only be a command name. If you need to set some specific environment, you should put a wrapper shell script.

Reply:
Yeah, this is almost certainly not needed either, just left over from the Fluffychat desktop file. Removed.